### DISCUSSION TITLE
Προσθήκη post του erasmus+ layout προς το netlify sitegr

### DISCUSSION THREAD

**Discussion by nkanagno** [ioniodi/sitegr#423] [Q&A] [answered]
Καλησπέρα σας @epidrome, έφτιαξα ήδη ένα layout το οποίο είναι το ενημερωτικό πρόγραμμα erasmus+ από την κεντρική ιστοσελίδα https://di.ionio.gr/gr/students/erasmus/ προς το netlify sitegr και παραδίδω το demo link μου απο κάτω και ήθελα να ρωτήσω αν αυτό που έχω κάνει θεωρείτε αποδεχτό για αυτό το παραδοτέο, πρωτού κάνω το issue μου:

## [demo](https://nkanagno-sitegr.netlify.app/erasmus/)

# links με τον πηγαίος κώδικας html-liquid που έγραψα:
### minimal-ionio (_includes):
[erasmus.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus.html)
[erasmus-table1.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table1.html)
[erasmus-table2.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table2.html)
[erasmus-download-links.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-download-links.html)

### sitegr:
[erasmus.html](https://github.com/nkanagno/sitegr/blob/demo-branch/_pages/erasmus.html)
#### Οι Αλλαγές που έγιναν στο navigation.yml: 
[navigation.yml](https://github.com/nkanagno/sitegr/blob/demo-branch/_data/navigation.yml)

**Comment by epidrome** [comment, marked as answer]
έχει κάπου liquid? χρησιμοποιεί τα δεδομένα που είναι αποθηκευμένα σε yaml?

αν είναι απλά μια στατική σελίδα σίγουρα δεν είναι αποδεκτό για 2ο παραδοτέο, μόνο ως 1ο

δες την τεκμηρίωση και σχετικές συζητήσεις δεν είναι εύκολος ο εντοπισμός του 2ου παραδοτέου

**Reply by nkanagno** [reply]
@epidrome 
Χρησιμοποιώ liquid γιατι έχω ουσιαστικά κάνει assign τα δεδομένα του κάθε πίνακα σε ένα μεγάλο string και με την liquid τα διαχωρίζω κάνοντας τα split 2 φορές (μία για τις γραμμές με "|" και ξανα με "," για το κάθε δεδομένο της κάθε γραμμής. Έπειτα τα εμφανίζω σε table με 2 εμφωλευμένα for loops της liquid. 
Παραθέτω από κάτω τα δύο αρχεία στα οποία το έχω κάνει:
[erasmus-table1.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table1.html)
[erasmus-table2.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table2.html)

**Reply by epidrome** [reply]
αν κατάλαβα σωστά προτείνεις να προσθέσουμε μια νέα ιδιότητα σε κάθε μάθημα που (δεν) είναι διαθέσιμο για erasmus? 

αυτό νομίζω πως είναι δεκτό

**Comment by nkanagno** [comment]
Επιτρέπεται να δημιουργήσω δικό μου αρχείο `.yml` όπως αυτό [`erasmus.yml`](https://github.com/nkanagno/sitegr/blob/demo-branch/_data/erasmus.yml) μέσα στον φάκελο `_data` του sitegr και να αποθηκεύσω εκεί τα δεδομένα του κάθε πίνακά μου? @epidrome 

Νέα αρχεία (με liquid-html που χρησιμοποιούν τα δεδομένα του [`erasmus.yml`](https://github.com/nkanagno/sitegr/blob/demo-branch/_data/erasmus.yml)):
[erasmus-table1.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table1.html)
[erasmus-table2.html](https://github.com/nkanagno/minimal-ionio/blob/master/_includes/erasmus-table2.html)
